alibi_detect/cd/mmd.py

```diff
@@ -88,9 +94,21 @@ def __init__(
 
         if backend == 'tensorflow' and has_tensorflow:
             kwargs.pop('device', None)
-            self._detector = MMDDriftTF(*args, **kwargs)  # type: ignore
+            if estimator == 'quad':
+                self._detector = MMDDriftTF(*args, **kwargs)  # type: ignore
+            elif estimator == 'linear':
+                kwargs.pop('n_permutations', None)
```

Comment:
Best to clarify in the docstrings that `n_permutations` is not used for the *linear* estimator.

Reply:
Fixed.